From dc6afabbd94fa5b8e1da286ba387814e5df70ff1 Mon Sep 17 00:00:00 2001
From: Rob Crittenden <rcritten@redhat.com>
Date: Fri, 3 Feb 2017 08:06:14 -0500
Subject: [PATCH] Make an invalid protocol setting fatal

An invalid protocol setting would end up with the default
list which is a bit unexpected so kill the server and let
the admin figure out what went wrong and fix it.

Resolves BZ 1389114
---
 nss_engine_init.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/nss_engine_init.c b/nss_engine_init.c
index 03ac644..0bb2054 100644
--- a/nss_engine_init.c
+++ b/nss_engine_init.c
@@ -785,6 +785,7 @@ static void nss_init_ctx_protocol(server_rec *s,
                     ap_log_error(APLOG_MARK, APLOG_WARNING, 0, s,
                                  "%s:  Unknown protocol '%s' not supported",
                                  protocol_marker, token);
+                    nss_die();
                 }
             }
         }
-- 
2.9.3

